source/fundamentals/crud.txt

```diff
@@ -1,13 +1,26 @@
-
-====
-CRUD
-====
+===============
+CRUD Operations
+===============
 
 .. default-domain:: mongodb
 
 .. toctree::
-   :titlesonly:
-   :maxdepth: 1
+   :caption: CRUD Operations
+
+   /fundamentals/crud/read-operations
+   /fundamentals/crud/write-operations
+
+CRUD (Create, Read, Update, Delete) operations allow you to work with
+the data stored in MongoDB.
+
+The CRUD operation documentation is categorized in two sections:
+
+- :doc:`Read Operations </fundamentals/crud/read-operations>` find and return
+  documents stored within your MongoDB database.
```

Comment:
Nitpick:
Not certain, but I think we usually talk about data stored "in" a database rather than within. In either case, it would be good to stay consistent (see line 21).